site/docs/get-started/vagrant.md

```diff
@@ -19,10 +19,10 @@ Additional details on how to configure the system properly can be found in the l
 
 ### Step 2. Download and deploy the appliance
 
-1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/vagrant) and run it
+1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/ManageIQ) and run it
```

Comment:
updated so now is lowercase. The actual vagrant init has to be as it is or it won't work (it complains saying the the  actual name has some uppercase and I am trying to download it lowercase)